bin/import-vsp-certs

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -e
+
+if [ $# -lt 2 ]; then
+  echo "Usage: $0 ENVIRONMENT_NAME VERSION_NUMBER"
+  exit 1
+fi
+
+ENVIRONMENT_NAME=$1
+VERSION_NUMBER=$2
+
+case $ENVIRONMENT_NAME in
+  "development")
+    ENVIRONMENT_SHORT_NAME="Dev"
+    AZURE_SUBSCRIPTION_ID="8655985a-2f87-44d7-a541-0be9a8c2779d"
+    KEY_VAULT_NAME="s118d01-secrets-kv"
+    ;;
+  "production")
+    ENVIRONMENT_SHORT_NAME="Prod"
+    AZURE_SUBSCRIPTION_ID="88bd392f-df19-458b-a100-22b4429060ed"
+    KEY_VAULT_NAME="s118p01-secrets-kv"
+    ;;
+  *)
+    echo "Could not find an known environment with the name: $ENVIRONMENT_NAME"
+    exit 1
+    ;;
+esac
+
+function check_existing_key {
+  local FILE_PREFIX=$1
+
+  echo "Attempting to fetch existing key..."
+
+  if ! az keyvault secret show \
+    --vault-name $KEY_VAULT_NAME \
+    --name "${FILE_PREFIX}Key" \
+    > /dev/null
+  then
+    KEY_EXISTED=
+  else
+    echo "Key found."
+
+    KEY_EXISTED=1
+  fi
+}
+
+function check_existing_cert {
+  local FILE_PREFIX=$1
+
+  echo "Attempting to fetch existing certificate..."
+
+  if ! az keyvault secret show \
+    --vault-name $KEY_VAULT_NAME \
+    --name "${FILE_PREFIX}Certificate" \
+    > /dev/null
+  then
+    CERT_EXISTED=
+  else
+    echo "Certificate found."
+
+    CERT_EXISTED=1
+  fi
+}
+
+function store_certificate {
+  local FILE_PREFIX=$1
+  local CERTIFICATE=$2
+
+  if ! [ "$CERT_EXISTED" ]; then
+    az keyvault secret set \
+      --vault-name $KEY_VAULT_NAME \
+      --name "${FILE_PREFIX}Certificate" \
+      --value "$CERTIFICATE"
```

Comment:
Does this work OK with keyvault? I had to use the `--file` option as the linebreaks got mangled if I set a value

Reply:
Yes, it does. Just confirmed by looking at what's in the Key Vault, imported by this script.

Reply:
🆒